source/fundamentals/crud/sort.txt

```diff
@@ -0,0 +1,110 @@
+============
+Sort Results
+============
+
+.. default-domain:: mongodb
+
+Use ``sort`` to change the order in which read operations return
+documents. The ``sort`` function tells MongoDB to order returned documents by the
+values of one or more fields in a certain direction. To sort returned
+documents by a field in ascending (lowest first) order, use the ``Sorts.ascending``
+function. To sort in descending (greatest first) order instead, use ``Sorts.descending``.
+If you do not specify a sort, MongoDB does not guarantee the order of
+query results.
+
+Follow the instructions in the examples below to insert data into
+a collection and perform a sort on the results of a query.
+Consider a collection containing documents that describe books. To
+insert this data into a collection, run the following operation:
+
+.. code-block:: java
+
+   collection.insertMany([
+        new Document().append("_id", 1)
+            .append("title", "The Brothers Karamazov").append("length", 824).append("author", "Dostoyevksy"),
+        new Document().append("_id", 2)
+            .append("title", "Les Misérables").append("length", 1462).append("author", "Hugo"),
+        new Document().append("_id", 3)
+            .append("title", "Atlas Shrugged).append("length", 1088).append("author", "Rand"),
```

Comment:
```suggestion
            .append("title", "Atlas Shrugged").append("length", 1088).append("author", "Rand"),
```
The syntax highlighting for this code block looked absolutely crazy to me, until I realized that there was a missing quote 🤭